### PR TITLE
Make isoline credentials independent

### LINKED ISSUE #128: Add credentials for isolines as new service
Right now we have the credentials for the isolines services shared with the HERE geocoder credentials.

Now we have to split it up in two different configurations: one for HERE geocoder and other for HERE isolines

### PR DESCRIPTION
**Pull request by iriberri**
Closes #128 

cc @ethervoid

### CODE REVIEW

**Comment by ethervoid** [comment]
To fix the tests change the 00_install_test and add the new configuration

**Comment by iriberri** [comment]
@ethervoid hm, it seems it's still failing with the same errors with https://github.com/CartoDB/dataservices-api/pull/130/commits/5f74d012a6950c7a55b7b23cc338c79061981e9e

**Comment by ethervoid** [comment]
You have to change this tests too https://github.com/CartoDB/dataservices-api/blob/master/server/lib/python/cartodb_services/test/test_helper.py#L46

**Comment by iriberri** [comment]
````
/server/extension/test/results/00_install_test.out	2016-03-31 14:45:50.614790605 +0000
***************
*** 1,6 ****
--- 1,7 ----
  -- Install dependencies
  CREATE EXTENSION postgis;
  CREATE EXTENSION schema_triggers;
+ ERROR:  could not open extension control file "/usr/share/postgresql/9.3/extension/schema_triggers.control": No such file or directory
  CREATE EXTENSION plpythonu;
  CREATE EXTENSION cartodb;
  CREATE EXTENSION cdb_geocoder;

======================================================================
````

**Comment by iriberri** [comment]
Tested in staging: the keys seem to work appropriately. 
* Breaking geocoder credentials make geocoding calls fail but isolines work.  :+1:
* Breaking isolines credentials makes geocoding calls work but isolines fail. :+1:
* Breaking both makes everything fail. :+1:
* The correct credentials for both are also working. :+1:

**Comment by iriberri** [comment]
:boom: Integration tests passed!

File: server/lib/python/cartodb_services/setup.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 setup(
     name='cartodb_services',
 
-    version='0.4.0',
+    version='0.4.1',
```
Comment (ethervoid) [inline server/lib/python/cartodb_services/setup.py]: Change it to 0.4.2 because I've to make a hotfix

**Comment by iriberri** [comment]
Done, @ethervoid!

**Pull request merged**